Add firmware packages to profiles

commonsourcecs opened the pull request:
Adding the various `linux-firmware-*` packages to the relevant chwd profiles seems like a good idea.

This would also (begin to) address #185

ventureoo commented on profiles/pci/graphic_drivers/profiles.toml:
```diff
@@ -22,7 +22,7 @@ class_ids = "0300 0302 0380"
 device_ids = "*"
 priority = 10
-packages = 'nvidia-utils egl-wayland nvidia-settings opencl-nvidia lib32-opencl-nvidia lib32-nvidia-utils libva-nvidia-driver vulkan-icd-loader lib32-vulkan-icd-loader'
+packages = 'nvidia-utils egl-wayland nvidia-settings opencl-nvidia lib32-opencl-nvidia lib32-nvidia-utils libva-nvidia-driver vulkan-icd-loader lib32-vulkan-icd-loader linux-firmware-nvidia'
```
NVIDIA drivers don't need linux-firmware package. They ship GSP firmware themselves with nvidia-utils package. linux-firmware-nvidia package needed only for nouveau driver.

ventureoo commented:
I'm still not sure if it's good idea, since we're anyway installing linux-firmware meta-package in calamares.

commonsourcecs commented:
There would seem to be still rationale for usefulness in recovery/repair, converted (Arch, etc) installs, or any other function post-install. 

The branch and PR are now updated with the removal of linux-firmware-nvidia from nvidia profiles. Thank you for the correction.

commonsourcecs commented:
Sorry about that. Updated again.

ventureoo reviewed (approved):
LGTM
